dotcom-rendering/cypress/integration/parallel-1/article.elements.spec.js

```diff
@@ -148,6 +148,14 @@ describe('Elements', function () {
 		});
 
 		it('should render the interactive using a non-boot.js', function () {
+			cy.on('uncaught:exception', (err) => {
+				if (err.message.includes('window.require')) {
+					// This error is unrelated to the test in question so return  false to prevent
+					// this error from failing this test
+					return false;
+				}
+			});
```

Comment:
This error seemed to only start appearing in prod mode? I suspect this interactive wasn't loading before